CONTRIBUTING.md

```diff
@@ -125,16 +125,16 @@ If you are interested in hearing more, please contact one of our the maintainer.
 
 ### Pull requests
 
-Please follow these steps to have your contribution considered by the maintainers:
+Please follow these steps to have your contribution considered by the maintainers
 
 1. Follow the [styleguides](#styleguides)
 2. Before you submit your pull request run [pytests](https://pypi.org/project/pytest/) and make sure that all tests pass. In order to run pytest, run the following line from the top directory:
 
 ```bash
 make test
 ```
-
-3. After you submit your pull request, verify that all status checks are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+3. You can use this [template][pr-template] to create pull request.
```

Comment:
If the template is properly set up in Github, when you create a PR, the pull request is automatically populated with the template so this doesn't need to be mentioned.

Reply:
If we have `PULL_REQUEST_TEMPLATE.md` in the project root or `.github/PULL_REQUEST_TEMPLATE.md`, then it will automatically populate the PR with the contents of the file.
If we have a `PULL_REQUEST_TEMPLATE` folder with more specific pull request templates, to auto-populate the PR content, you will need to add a query parameter like so: https://github.com/my_project/compare/master…branch_to_merge?expand=1&template=bug-template.md

See https://www.gitkraken.com/blog/enhancing-pull-request-descriptions-templates for more info.

Reply:
Thanks for pointing it out. I was initially planned to make different templates and let users to query on URL, but it is probably sufficient to have a single version at this point.
I pushed the change directly, 987211c5e52a125b233e2a573f3aa467ee1d56b0

Reply:
Looks good! I tested it out--new PRs are autopopulated with content now :)